src/polyfill.js

```diff
@@ -70,6 +68,7 @@ function setup() {
         typeof sanitizerObj.getConfiguration !== "function"
       ) {
         sanitizerObj = new Sanitizer();
+        Object.freeze(sanitizerObj);
```

Comment:
This might be too janky, but maybe something like the following will work:

```suggestion
        sanitizerObj.prototype.constructor = sanitizer;
        Object.freeze(sanitizerObj);
```

Reference: https://github.com/mozilla/sanitizer-polyfill/issues/54#issuecomment-907610013